commands/keplr.js

```diff
@@ -34,7 +34,14 @@ const keplr = {
       extensionVersion,
     };
   },
-
+  async disconnectWalletFromDapp() {
+    await playwright.waitAndClickByText(
+      'Disconnect All',
+      playwright.keplrPermissionWindow(),
+      exact = false,
```

Comment:
dont think js actually reads the kwarg name in the function call
![image](https://github.com/agoric-labs/synpress/assets/8860764/fb33072f-b1ca-493f-9d00-ec8956a4876d)